crypto/objects/obj_dat.h

```diff
@@ -2288,9 +2288,10 @@ static const ASN1_OBJECT nid_objs[NUM_NID] = {
     {"SSHKDF", "sshkdf", NID_sshkdf},
     {"SM2-SM3", "SM2-with-SM3", NID_SM2_with_SM3, 8, &so[7766]},
     {"SSKDF", "sskdf", NID_sskdf},
+    {"X963KDF", "x963kdf", NID_x963kdf},
```

Comment:
I wonder if the name here should have a "dot" too? Not sure

Reply:
I assume it would be bad in the nid name though :)

Reply:
maybe use a _ for that case?

Reply:
The nid name is probably ok as is. Its just whether we add the "." in the sn/ln strings. I lean towards "yes"

Reply:
I might need to use dash as the nid name comes from the  same fields in objects.txt
# NID for X9.63-2001 KDF
                            : X963KDF            : x963kdf

Reply:
Hmm. Ok. Not sure which is better, so I'll leave it to you.

Reply:
I think it will probably be more confusing so I will leave it in a consistent form.

Reply:
I think adding the period is not a good thing to do; we always use x509 asn1, etc., and being consistent seems more important.  Folks who care about X9.63 will not be confused by x963.